### PR TITLE
Feat: added cart

### PR DESCRIPTION
**Pull request by anjali1102**
👉👉CHECK PREVIEW [HERE...](https://hogwarts-supplies-ef1bbjekm-anjali1102.vercel.app/pages/cart.html)

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
                   <button class="search-btn"><i class="fa fa-search"></i></button>
               </div>
               <a href="/pages/login.html"><img src="../assets/user.png" alt=""></a>
-              <a href="#"><img src="../assets/cart.png" alt=""></a>
+              <a href="/pages/cart.html"><img src="../assets/cart.png" alt=""></a>
```
Comment (ttahm3d) [inline index.html:21]: Give an alt text always.

File: pages/cart.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://rapidui.vercel.app/docs.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <link rel="stylesheet" href="../style/cart.css" />
+    <title>Login</title>
```
Comment (ttahm3d) [inline pages/cart.html:13]: Title is not matching the file name.

File: pages/cart.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+          <img
+            class="card-img horizontal-img"
+            src="/assets/feature12.jpg"
+            alt=" "
+          />
+          <div class="card-info">
+            <div class="card-title">
+              <div>
+                <h3>Specs</h3>
+              </div>
```
Comment (ttahm3d) [inline pages/cart.html:49]: This div can be avoided.

**Review by ttahm3d — COMMENTED** [review]
Added few comments please take a look.

**Review by shraddha15-creator — APPROVED** [review]

**Review by Husain01 — APPROVED** [review]
Looks good to go 🎉

**Pull request merged**